doc/release-notes.md

```diff
@@ -156,7 +221,29 @@ RPC Deprecations
 - `z_gettotalbalance` has been deprecated. Use `z_getbalanceforaccount` instead.
 - `dumpwallet` has been deprecated. Use `z_exportwallet` instead.
 
+Build System
+------------
+
+- Clang has been updated to use LLVM 13.0.1.
+- libcxx has been updated to use LLVM 13.0.1, except on Windows where it uses 13.0.0-3.
```

Comment:
```suggestion
- libc++ has been updated to use LLVM 13.0.1, except on Windows where it uses 13.0.0-3.
```
It's occasionally called `libcxx` (and [somewhat inconsistently](https://github.com/zcash/zcash/blob/master/depends/packages/libcxx.mk#L26-L27)) in filenames, [etc.](https://lists.llvm.org/mailman/listinfo/libcxx-commits), but the official name is libc++.

Reply:
I will make this correction in the release PR.